v3-docs/docs/tutorials/meteorjs-vue.md

```diff
@@ -0,0 +1,1375 @@
+
+
+::: code-group
+```vue [imports/ui/App.vue]
+<script setup>
+import Task from './components/Task.vue'
+
+const getTasks = () => {
+    return [
+      { _id: 1, text: 'Install Node.js 20' },
+      { _id: 2, text: 'Install Meteor 3' },
+      { _id: 3, text: 'Create and run your project' },
+    ];
+}
+</script>
+
+<template>
+  <div class="p-8">
+    <h1>Hello Meteor</h1>
+  </div>
+</template>
+```
+:::
+
+### 1.5: Rendering Tasks
+
+
+Now let's add some simple rendering logic with Vue to display our list items using the `Task` component.
+
+Update the template of the `App` component to include the list of tasks:
+
+
+::: code-group
+```vue [imports/ui/App.vue]
+<script setup>
+import Task from './components/Task.vue'
+
+const getTasks = () => {
+    return [
+      { _id: 1, text: 'Install Node.js 20' },
+      { _id: 2, text: 'Install Meteor 3' },
+      { _id: 3, text: 'Create and run your project' },
+    ];
+}
+</script>
+<template>
+  <div class="container">
+    <header>
+      <h1 class="text-4xl font-bold text-gray-800 my-4">Todo List</h1>
+    </header>
+    <ul class="list-disc list-inside p-4">
+      <Task v-for="task in getTasks()" :key="task._id" :task="task" />
+    </ul>
+  </div>
+</template>
+```
+:::
+
+For more information on Vue iterations, you can read [here](https://vuejs.org/api/built-in-directives.html#v-for).
+
+In the next step, we will connect to the MongoDB database to store our tasks.
+
+
+## 2: Collections
+
+
+Meteor already sets up MongoDB for you. In order to use our database we need to create a *collection*, which is where we will store our *documents*, in our case our `tasks`.
+You can read more about collections [here](http://guide.meteor.com/collections.html).
+
+
+In this step we will implement all the necessary code to have a basic collection for our tasks up and running.
+
+### 2.1: Create Tasks Collection
+
+
+Before creating our collection, let's remove the `links.js` file from the `imports/api`  folder because we won't use it. Now, you can create a new collection to store our tasks by creating a new file named `tasksCollection.js` inside the `imports/api` folder.
+
+
+::: code-group
+```javascript [imports/api/tasksCollection.js]
+import { Mongo } from 'meteor/mongo';
+ 
+export const TasksCollection = new Mongo.Collection('tasks');
+```
+:::
+
+The code above instantiates a new MongoDB collection and exports it. You can read more about app structure and imports/exports [here](https://guide.meteor.com/structure.html).
+
+### 2.2: Initialize Tasks Collection
+
+
+To make our collection work, you need to import it on the server to set things up.
+
+You can use import `'./imports/api/tasksCollection'` or `import { TasksCollection } from './imports/api/tasksCollection'` if you plan to use `TasksCollection` in the same file. Just make sure it's imported.
```

Comment:
```suggestion
You can use `import './imports/api/tasksCollection'` or `import { TasksCollection } from './imports/api/tasksCollection'` if you plan to use `TasksCollection` in the same file. Just make sure it's imported.
```

Anyway, here, I would prefer to suggest them to use `import { TasksCollection } from './imports/api/tasksCollection'`, since import `import './imports/api/tasksCollection'` imports eagerly all modules, and could be an antipattern towards tree-shakeable code, since it may introduce side effects.

Reply:
Do you recommend changing it, or should I leave it as is? @nachocodoner